The test case assumes that the system makefiles were already installed,
however, since Alpine tests are run before the package is installed this
is not the case. We can fix this by specifying a different path to the
system makefiles.

diff -upr bmake.orig/unit-tests/varname-makeflags.mk bmake/unit-tests/varname-makeflags.mk
--- bmake.orig/unit-tests/varname-makeflags.mk	2022-01-16 16:38:51.465605125 +0100
+++ bmake/unit-tests/varname-makeflags.mk	2022-01-16 16:39:08.939020513 +0100
@@ -35,7 +35,7 @@ OUTPUT!=	env MAKEFLAGS=ikrs ${MAKE} -f /
 # variable MAKEFLAGS, it is no longer split.  In this example, the word
 # "d0ikrs" is treated as a target, but the option '-v' prevents any targets
 # from being built.
-OUTPUT!=	env MAKEFLAGS=d0ikrs ${MAKE} -f /dev/null -v .MAKEFLAGS
+OUTPUT!=	env MAKESYSPATH=../mk MAKEFLAGS=d0ikrs ${MAKE} -f /dev/null -v .MAKEFLAGS
 .if ${OUTPUT} != " -V .MAKEFLAGS"
 .  error ${OUTPUT}
 .endif
